jobs/urls.py

```diff
@@ -1,18 +1,19 @@
 from django.urls import path
 
-from .views import (index, pricing, profile, about, BusinessCreateView, 
+from .views import (index, pricing, profile, about, BusinessCreateView,
                     FreelancerCreateView, FreelancerUpdateView,
-                    FreelancerDetailView, FreelancerListView, handle_login)
+                    FreelancerDetailView, handle_login, list_developers,)
 
 urlpatterns = [
     path('', index, name='home'),
     path('profile/', profile, name='profile'),
     path('about/', about, name='about'),
     path('pricing/', pricing, name='pricing'),
-    path('developers/', FreelancerListView.as_view(), name='freelancer-list'),
+    # path('developers/', FreelancerListView.as_view(), name='freelancer-list'),
     path('account-setup/', handle_login, name='handle-login'),
     path('developer/<int:pk>/', FreelancerDetailView.as_view(), name='freelancer-detail'),
     path('developer/create/', FreelancerCreateView.as_view(), name="freelancer-create"),
     path('developer/update/', FreelancerUpdateView.as_view(), name="freelancer-update"),
     path('business/create/', BusinessCreateView.as_view(), name="business-create"),
-]
\ No newline at end of file
+    path('list-developers/',list_developers,name="list_developers"),
```

Comment:
lets keep the path 'developer/' & use '-' instead of '_' for names
```suggestion
    path('developers/',list_developers,name="list-developers"),
```